### PR TITLE
fix(Marshaller): Open new DDL should auto-select first element

### PR DESCRIPTION
**Pull request by GordonSmith**
Fixes GH-2933

Signed-off-by: Gordon Smith <gordonjsmith@gmail.com>

### CODE REVIEW

File: packages/marshaller/src/dashy.ts [hunk 1]
```diff
@@ -146,6 +146,10 @@ export class Dashy extends SplitPanel {
         this._elementContainer.clear();
         this._dashboard.restore(json);
         return this._dashboard.renderPromise().then(() => {
+            const elements = this._elementContainer.elements();
+            if (elements.length) {
+                this._dashboard.activate(elements[0]);
+            }
```
Comment (jbrundage) [inline packages/marshaller/src/dashy.ts]: I'm not exactly sure what's going on here, but it doesn't seem to be related to "Enhanced classed call to match d3 v4 call signature"

Comment (GordonSmith) [reply packages/marshaller/src/dashy.ts]: In d3 v4 the classed method can only be called like this:
Setting:
```
element.classed("someClassID", true);
```
Getting:
```
element.classed("someClassID");
```
In v3 it also supported:
Setting:
```
element.classed({
    "someClassID": true,
    "someOtherClassID": false
});
```
Which is the variant we supported.

This PR simply adds the ability to set / get both ways.

Comment (jbrundage) [reply packages/marshaller/src/dashy.ts]: I understood that about this original PR, but it seems like you may have overwritten that PR with this commit? "fix(Marshaller): Open new DDL should auto-select first element"

Comment (GordonSmith) [reply packages/marshaller/src/dashy.ts]: No idea what happened there - but the title is now corrected (and I removed the other confusing comments!)
